### PR TITLE
fix: node-title-is-overflow-in-checklist

### LINKED ISSUE #5900: node title is overflow in checklist
### Self Checks

- [X] This is only for bug report, if you would like to ask a question, please head to [Discussions](https://github.com/langgenius/dify/discussions/categories/general).
- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Dify version

0.6.11

### Cloud or Self Hosted

Cloud, Self Hosted (Docker), Self Hosted (Source)

### Steps to reproduce

node title is overflow in checklist 

### ✔️ Expected Behavior

node title is  truncate in checklist 

### ❌ Actual Behavior

node title is overflow in checklist 
image
image

### PR DESCRIPTION
**Pull request by aixgeek**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes https://github.com/langgenius/dify/issues/5900

## Type of Change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update, included: [Dify Document](https://github.com/langgenius/dify-docs)
- [ ] Improvement, including but not limited to code refactoring, performance optimization, and UI/UX improvement
- [ ] Dependency upgrade

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] TODO

# Suggested Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] I ran `dev/reformat`(backend) and `cd web && npx lint-staged`(frontend) to appease the lint gods
- [ ] `optional` I have made corresponding changes to the documentation 
- [ ] `optional` I have added tests that prove my fix is effective or that my feature works
- [ ] `optional` New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by aixgeek** [comment]
image

image

**Comment by crazywoola** [comment]
Thanks for your contribution. Please open an issue before creating a PR or link to an existing issue, fix the errors in CI as well.

**Comment by aixgeek** [comment]
#5900 Add issure @crazywoola

**Comment by crazywoola** [comment]
Please resolve lint errors in the CI.

**Comment by aixgeek** [comment]
> Please resolve lint errors in the CI.

done

**Comment by aixgeek** [comment]
@iamjoel

File: web/app/components/workflow/header/checklist.tsx [hunk 1]
```diff
@@ -122,7 +122,9 @@ const WorkflowChecklist = ({
                               className='mr-1.5'
                               toolIcon={node.toolIcon}
                             />
-                            {node.title}
+                            <span className='w-[340px] truncate'>
```
Comment (iamjoel) [inline web/app/components/workflow/header/checklist.tsx]: You should use 'grow' instead of 'w-[340px]' to increase flexibility.

**Review by iamjoel — CHANGES_REQUESTED** [review]

**Review by iamjoel — APPROVED** [review]

**Pull request merged**